handleData.js

```diff
@@ -186,6 +185,8 @@ class Character extends Player {
             "Platinum"   : 0,
             "Electrum"   : 0,
         };
+        this.currentExperience = initialExperience;
+        this.experienceRequirements = [300, 900, 2700, 5400];
```

Comment:
These should be constants somewhere else, these numbers aren't specific to a given character